### PR TITLE
fix gf to be like issue #2683

### PR DESCRIPTION
**Pull request by SuyogSoti**
**What this PR does / why we need it**:
fix gf to be like issue #2683

**Which issue(s) this PR fixes**
#2683 

**Special notes for your reviewer**:

### CODE REVIEW

File: src/cmd_line/commands/file.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ export interface IFileCommandArguments extends node.ICommandArgs {
   bang?: boolean;
   position?: FilePosition;
   lineNumber?: number;
+  dontCreateFile?: boolean;
```
Comment (jpoon) [inline src/cmd_line/commands/file.ts]: can you do the negation and rename something more descriptive like `createFileIfNotExists`

File: src/cmd_line/commands/file.ts [hunk 2]
```diff
@@ -102,7 +103,12 @@ export class FileCommand extends node.CommandBase {
             filePath = pathWithExt;
           } else {
             // create file
-            fs.closeSync(fs.openSync(filePath, 'w'));
+            if (this.arguments.dontCreateFile === true) {
+              vscode.window.showErrorMessage('File path does not exits');
```
Comment (jpoon) [inline src/cmd_line/commands/file.ts]: there's a function in `util.ts` for this, can you use that? Also output the filepath.

File: src/cmd_line/commands/file.ts (hunk 1 above)
Comment (jpoon) [reply src/cmd_line/commands/file.ts]: Thanks so much for this PR! <3

File: src/actions/commands/actions.ts [hunk 3]
```diff
@@ -2346,7 +2346,7 @@ class CommandOpenFile extends BaseCommand {
       const fileCommand = new FileCommand({
         name: filePath,
         lineNumber: lineNumber,
-        dontCreateFile: true,
+        createFileIfNotExists: false,
```
Comment (jpoon) [inline src/actions/commands/actions.ts]: After changing the negation, the default value is 'false'. Which means you'll need to update the other spot (`:e`) to set this value to `true`.

**Comment by xconverge** [comment]
@jpoon looks like you can finish your review here now if you get a chance!

**Comment by SuyogSoti** [comment]
yes thanks!

**Comment by jpoon** [comment]
Thanks for your contribution @SuyogSoti !

**Pull request merged**